src/st_stuff.c

```diff
@@ -851,6 +851,14 @@ void ST_drawWidgets(boolean refresh)
   int i;
   int maxammo = plyr->maxammo[weaponinfo[w_ready.data].ammo];
 
+  if (!crispy_hud)
```

Comment:
`crispy_hud` or `st_crispyhud`?

Reply:
Right, fixed in https://github.com/fabiangreffrath/woof/pull/691/commits/05d6989c875356d2410d11c9ce7b9e356dfa165d